From f3f3aeac5063eb7cc1ebf1c459db15c9566b0a4a Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 8 May 2012 17:12:25 +0300
Subject: [PATCH 157/311] conf/obj.c: add a "DLD-wide" state specification

New State Specification section summarizes all things that have state.

LogD: #1156
RB: r/714
---
 conf/obj.c | 8 ++++++++
 conf/obj.h | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/conf/obj.c b/conf/obj.c
index 7b8eeff..419b046 100644
--- a/conf/obj.c
+++ b/conf/obj.c
@@ -181,6 +181,7 @@
  * - @ref conf_dlspec_objops
  * - @ref conf_dlspec_map
  * - @subpage confd-lspec
+ * - @ref conf-lspec-state
  *
  * <!---------------------------------------------------------------->
  * @subsection conf-lspec-comps Components Overview
@@ -203,6 +204,13 @@
  * confc, confd loads the necessary data from the configuration
  * database and updates the cache.
  *
+ * <!---------------------------------------------------------------->
+ * @subsection conf-lspec-state State Specification
+ *
+ * - @ref conf-fspec-obj-status
+ * - @ref confc-lspec-state "States of a confc state machine"
+ * - @ref confd-lspec-state "States of a FOM at confd side"
+ *
  * <hr> <!------------------------------------------------------------>
  * @section conf-conformance Conformance
  *
diff --git a/conf/obj.h b/conf/obj.h
index 827ebbe..62b4195 100644
--- a/conf/obj.h
+++ b/conf/obj.h
@@ -61,7 +61,7 @@
  * configuration objects.
  *
  * <hr> <!------------------------------------------------------------>
- * @section conf-fspec-obj-status Configuration Object States
+ * @section conf-fspec-obj-status Configuration Object Status
  *
  * A configuration object exists in one of three states:
  *   - C2_CS_MISSING --- configuration is absent and is not being
-- 
1.8.3.2

